sdk/release/RELEASE.md

```diff
@@ -294,14 +296,14 @@ windows` inside the `daml-language-ad-hoc` project. This command prints IP
 
 1. Close VS Code.
 
-> Note: when running `daml studio --replace=always`, you force the installation
-> of the VSCode extension bundled with the Daml SDK, and _disable the
-> auto-upgrade mechanism in VSCode_. To instruct VSCode to go back to the
-> published version of the extension, including auto-upgrades, you can run
->
-> ```
-> daml studio --replace=published
-> ```
+   > Note: when running `daml studio --replace=always`, you force the installation
+   > of the VSCode extension bundled with the Daml SDK, and _disable the
+   > auto-upgrade mechanism in VSCode_. To instruct VSCode to go back to the
+   > published version of the extension, including auto-upgrades, you can run
+   >
+   > ```
+   > daml studio --replace=published
+   > ```
```

Comment:
While we're at it, this note would probably best be placed inside of step 11 (Run `daml studio --replace=always`) if it's going inside a step, rather than onto step 18 (Close VS Code)